tests/js/user-tests.js

```diff
@@ -150,6 +150,21 @@ module.exports = {
     TestCase.assertEqual(err.message, "function not found: 'error'");
   },
 
+  async testRemoteMongoClient() {
+    let app = new Realm.App(appConfig);
```

Comment:
Not really testing much, but I guess that's fine for now, we should reuse the Realm Web integration tests long term.

Reply:
Yeah, the tests were just to ensure that everything was hooked up right and working sanely. Agree that we should one have one comprehensive test long term.